src/containers/HSM/TemplateLibraryModal/TemplateLibraryModal.module.css

```
@@ -0,0 +1,220 @@
+  text-align: center;
+  padding: 0 var(--app-space-3xl);
+}
+
+/* the app's actual brand-primary green (same token used for buttons/nav
+   elsewhere), not the Simulator's own muted WhatsApp-mimicking header color —
+   pulled out to the box's edges with negative margins so it reads as a
+   colored strip, while the rest of `.PreviewBox`'s padding still applies to
+   the bubble below. The negative top/left/right offsets must match
+   `.PreviewBox`'s own padding exactly for that "pulled to the edge" trick to
+   work, so they're expressed as the same token, negated. */
+.PreviewBox :global([class*='PreviewHeader']) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)